### PR TITLE
zh: resync scheduling files

### PR DESCRIPTION
**Pull request by chenrui333**
part of #27672

### CODE REVIEW

File: content/zh/docs/concepts/scheduling-eviction/_index.md [hunk 1]
```diff
@@ -1,5 +1,55 @@
...
+-->
+
+<!--
+In Kubernetes, scheduling refers to making sure that {{<glossary_tooltip text="Pods" term_id="pod">}}
+are matched to {{<glossary_tooltip text="Nodes" term_id="node">}} so that the
+{{<glossary_tooltip text="kubelet" term_id="kubelet">}} can run them. Preemption
+is the process of terminating Pods with lower {{<glossary_tooltip text="Priority" term_id="pod-priority">}}
+so that Pods with higher Priority can schedule on Nodes. Eviction is the process
+of terminating one or more Pods on Nodes.
+-->
```
Comment (chenrui333) [inline content/zh/docs/concepts/scheduling-eviction/_index.md:33]: Did not quite translate this section as I feel like it overlaps with the description part above?

File: content/zh/docs/concepts/scheduling-eviction/assign-pod-node.md [hunk 2]
```diff
@@ -136,8 +136,18 @@ with a standard set of labels. See [Well-Known Labels, Annotations and Taints](/
 -->
 ## 插曲：内置的节点标签 {#built-in-node-labels}
 
-除了你[添加](#attach-labels-to-node)的标签外，节点还预先填充了一组标准标签。
-参见[常用标签、注解和污点](/zh/docs/reference/labels-annotations-taints/)。
+除了你[添加](#step-one-attach-label-to-the-node)的标签外，节点还预制了一组标准标签。
```
Comment (chenrui333) [inline content/zh/docs/concepts/scheduling-eviction/assign-pod-node.md:139]: I dont know how we are dealing these inline heading cross-references, but it got updated anyway.

File: content/zh/docs/concepts/scheduling-eviction/pod-overhead.md [hunk 3]
```diff
@@ -299,7 +311,7 @@ kube-state-metrics, but is expected in a following release. Users will need to b
 from source in the meantime.
 -->
 在 [kube-state-metrics](https://github.com/kubernetes/kube-state-metrics) 中可以通过 `kube_pod_overhead` 指标来协助确定何时使用 PodOverhead 以及协助观察以一个既定开销运行的工作负载的稳定性。
-该特性在 kube-state-metrics 的 1.9 发行版本中不可用，不过预计将在后续版本中发布。在此之前，用户需要从源代码构建 kube-state-metrics. 
+该特性在 kube-state-metrics 的 1.9 发行版本中不可用，不过预计将在后续版本中发布。在此之前，用户需要从源代码构建 kube-state-metrics.
```
Comment (tengqm) [inline content/zh/docs/concepts/scheduling-eviction/pod-overhead.md:314]:
```suggestion
该特性在 kube-state-metrics 的 1.9 发行版本中不可用，不过预计将在后续版本中发布。
在此之前，用户需要从源代码构建 kube-state-metrics.

```

File: content/zh/docs/concepts/scheduling-eviction/scheduler-perf-tuning.md [hunk 4]
```diff
@@ -77,17 +79,17 @@ had set a value of 100.
...
 [kube-scheduler configuration file](/docs/reference/config-api/kube-scheduler-config.v1beta1/)
 and then restart the scheduler.
-In many cases, the configuration file can be found at `/etc/kubernetes/config/kube-scheduler.yaml`. 
--->
-要修改这个值，编辑 [kube-scheduler 的配置文件](/zh/docs/reference/config-api/kube-scheduler-config.v1beta1/)，
-之后重启调度器。
-在很多场合下，配置文件位于 `/etc/kubernetes/config/kube-scheduler.yaml`。
+In many cases, the configuration file can be found at `/etc/kubernetes/config/kube-scheduler.yaml`
+ -->
+要修改这个值，先编辑 [kube-scheduler 的配置文件](/docs/reference/config-api/kube-scheduler-config.v1beta1/)
```
Comment (tengqm) [inline content/zh/docs/concepts/scheduling-eviction/scheduler-perf-tuning.md:88]:
```suggestion
要修改这个值，先编辑 [kube-scheduler 的配置文件](/zh/docs/reference/config-api/kube-scheduler-config.v1beta1/)
```

File: content/zh/docs/concepts/scheduling-eviction/scheduler-perf-tuning.md [hunk 5]
```diff
@@ -189,15 +191,21 @@ percentageOfNodesToScore: 50
...
+<<<<<<< HEAD
 In clusters with less than 50 feasible nodes, the scheduler still
 checks all the nodes because there are not enough feasible nodes to stop
-the scheduler's search early. 
+the scheduler's search early.
+=======
+{{< note >}} In clusters with less than 50 feasible nodes, the scheduler still
+checks all the nodes because there are not enough feasible nodes to stop
+the scheduler's search early.
+>>>>>>> f5d4f148f (zh: sync concepts/scheduling-eviction/scheduler-perf-tuning)
```
Comment (tengqm) [inline content/zh/docs/concepts/scheduling-eviction/scheduler-perf-tuning.md:208]: 这儿有问题

File: content/zh/docs/concepts/scheduling-eviction/scheduler-perf-tuning.md [hunk 6]
```diff
@@ -294,8 +302,16 @@ After going over all the Nodes, it goes back to Node 1.
...
 
 * 查阅 [kube-scheduler 配置参考 (v1beta1)](/zh/docs/reference/config-api/kube-scheduler-config.v1beta1/)
 
+=======
+## {{% heading "whatsnext" %}}
+
+<!-- * Check the [kube-scheduler configuration reference (v1beta1)](/docs/reference/config-api/kube-scheduler-config.v1beta1/) -->
+
+* 参见 [kube-scheduler 配置参考 (v1beta1)](/docs/reference/config-api/kube-scheduler-config.v1beta1/)
+>>>>>>> f5d4f148f (zh: sync concepts/scheduling-eviction/scheduler-perf-tuning)
```
Comment (tengqm) [inline content/zh/docs/concepts/scheduling-eviction/scheduler-perf-tuning.md:317]: 这里有冲突问题

File: content/zh/docs/concepts/scheduling-eviction/scheduler-perf-tuning.md [hunk 7]
```diff
@@ -294,8 +296,8 @@ After going over all the Nodes, it goes back to Node 1.
 -->
 在评估完所有 Node 后，将会返回到 Node 1，从头开始。
 
-
 ## {{% heading "whatsnext" %}}
 
-* 查阅 [kube-scheduler 配置参考 (v1beta1)](/zh/docs/reference/config-api/kube-scheduler-config.v1beta1/)
+<!-- * Check the [kube-scheduler configuration reference (v1beta1)](/docs/reference/config-api/kube-scheduler-config.v1beta1/) -->
 
+* 参见 [kube-scheduler 配置参考 (v1beta1)](/docs/reference/config-api/kube-scheduler-config.v1beta1/)
```
Comment (chenrui333) [inline content/zh/docs/concepts/scheduling-eviction/scheduler-perf-tuning.md:303]:
```suggestion
* 参见 [kube-scheduler 配置参考 (v1beta1)](/zh/docs/reference/config-api/kube-scheduler-config.v1beta1/)
```

File: content/zh/docs/concepts/scheduling-eviction/_index.md (hunk 1 above)
Comment (tengqm) [reply content/zh/docs/concepts/scheduling-eviction/_index.md:33]: please translate the above paragraph.

**Comment by tengqm** [comment]
Thanks.
/lgtm
/approve

**Pull request merged**